### PR TITLE
Data import improve logs

### PR DESCRIPTION
**Pull request by shamoh19**
Adding tooltip about the Datarow values.

Instead of clicking the open to see the row values, users can just hover to the messages to get information

Related to this [improvement](https://github.com/pimcore/data-importer/issues/111)

### CODE REVIEW

**Comment by mcop1** [comment]
@shamoh19 I tried to apply the patch, but after that the log tab is broken. You referred to another pr, but it was merged already. Did I miss something?

**Comment by shamoh19** [comment]
@mcop1 only these two PR's. But this is only adding the tooltip, I am not sure whether this is really breaking the log tab

File: bundles/AdminBundle/Resources/public/js/pimcore/log/admin.js [hunk 1]
```diff
@@ -193,6 +193,7 @@ pimcore.log.admin = Class.create({
                     flex: 220,
                     sortable: true,
                     renderer: function (s) {
+                        meta.tdAttr='data-qtip="'+s+'"';
```
Comment (jdreesen) [inline bundles/AdminBundle/Resources/public/js/pimcore/log/admin.js:196]: What's `meta`? Seems it isn't defined anywhere :confused:

Comment (shamoh19) [reply bundles/AdminBundle/Resources/public/js/pimcore/log/admin.js:196]: It's part of renderer function. It's been almost two months, I made this PR. It was working fine during that time. I have to check again.

May be we need to alter the function as below. 

renderer: function (s, metadata, record) {
                        meta.tdAttr='data-qtip="'+s+'"';
                        return Ext.util.Format.htmlEncode(s);
                    }

Comment (jdreesen) [reply bundles/AdminBundle/Resources/public/js/pimcore/log/admin.js:196]: yeah, seems like the parameter is missing

Comment (brusch) [reply bundles/AdminBundle/Resources/public/js/pimcore/log/admin.js:196]: @shamoh19 🏓 can you please recheck? Thanks!

**Comment by brusch** [comment]
Closed due to inactivity. Please feel free to continue the discussion or provide the missing information on this PR, we'll then reopen it if necessary. Thanks a lot!

**Pull request closed without merging**